zoo-css-challenge/style.css

```
@@ -166,18 +190,32 @@ footer {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 30px;
+
 }
 
 .education-container h3,
 .education-container p {
   margin-bottom: 1em;
 }
+.education-container button{
+ 
+  font-size: larger;
+
+}
+.education-container button:hover{
```

```suggestion
.education-container button:focus,
.education-container button:hover{
```
We should add a focus style too for keyboard users who use the `Tab` key to move to the button